src/framework/card/Card.jsx

```diff
@@ -1,52 +1,87 @@
 
-const Card = (props) => {
-  const wrapperStyle = {
-    height: props.height,
-    width: props.width,
-  };
+export default class Card extends Component {
 
-  return (
-    <Box classes="card" roundedCorners>
-      <div className="card__wrapper" style={wrapperStyle}>
-        <div
-          className="card__image"
-          style={{ backgroundImage: `url(${props.imageSrc})` }}
-        />
+  constructor(props) {
+    super(props);
+    this.wrapperStyle = {
+      height: props.height,
+      width: props.width,
+    };
+    this.dropdownItems = [
+      StatusDropdown.OPTIONS.PUBLISHED,
+      StatusDropdown.OPTIONS.UNPUBLISHED,
+    ];
 
-        <div className="card__info">
-          <Label>{props.title}</Label>
-          <SubLabel className="card__info__subLabel card__text--mediumgrey">
-            {props.subtitle}
-          </SubLabel>
-          <SubLabel className="card__info__subLabel card__text--lightgrey">
-            {props.description}
-          </SubLabel>
-        </div>
+    this.state = {
+      selectedOption: undefined,
+    };
+
+    this.onSelectOption = this.onSelectOption.bind(this);
+  }
+
+  onSelectOption(option) {
+    this.setState({
+      selectedOption: option,
+    });
+  }
 
-        <div className="card__footer">
-          <div className="card__footer--left">
-            <Tooltip message={props.tooltipText} width="150px">
-              <Label className="card__footer__label card__text--green">
-                {props.hightlightText}
-              </Label>
-            </Tooltip>
+  render() {
+    return (
+      <Box classes="card" roundedCorners>
+        <div className="card__wrapper" style={this.wrapperStyle}>
+          <div
+            className="card__image"
+            style={{ backgroundImage: `url(${this.props.imageSrc})` }}
+          />
+
+          <div className="card__info">
+            <Label>{this.props.title}</Label>
+            <SubLabel className="card__info__subLabel card__text--mediumgrey">
+              {this.props.subtitle}
+            </SubLabel>
+            <SubLabel className="card__info__subLabel card__text--lightgrey">
+              {this.props.description}
+            </SubLabel>
+          </div>
+
+          <div className="card__footer">
+            <div className="card__footer--left">
+              <div className="card__footer--left__tooltip">
+                <Tooltip message={this.props.tooltipText} width="150px">
+                  <Label className="card__footer__label card__text--green">
+                    {this.props.hightlightText}
+                  </Label>
+                </Tooltip>
+              </div>
+            </div>
+            <div className="card__footer--right">
+              <div className="card__footer--right__status">
+                <StatusDropdown
+                  onSelect={this.onSelectOption}
+                  options={this.dropdownItems}
+                  selectedOption={this.state.selectedOption}
```

Comment:
Maybe it's better to have the whole `StatusDropdown` as a prop passed into the component, and using `PropTypes.instanceOf` to only allow dropdown to be passed in as a prop. That way the card component don't need know anything about the callback, selectable values etc.